lib/js_of_ocaml/js_of_ocaml_stubs.c

```diff
@@ -1,5 +1,9 @@
 #include <caml/misc.h>
 
+void caml_bytes_of_array () {
+  caml_fatal_error("Unimplemented Javascript primitive caml_bytes_of_array!");
+}
+
```

Comment:
I’m a bit lost, can someone explain to me why using the API will not just raise this error?

Reply:
This is just to keep the Ocaml bytecode compiler happy. This code is executed instead:
https://github.com/ocsigen/js_of_ocaml/blob/a03f32c5a462b21ab84840fc4858632e1540004f/runtime/mlBytes.js#L512-L517

Reply:
Ah, I see, thanks!